source/beamlines.md

```diff
@@ -331,15 +342,54 @@ elements determines the order in which they should tracked through. For example,
 if a line contains the two zero length elements:
 ```{code} yaml
 BeamLine:
-  - kind: Line 
-    name: line
-  - markerA
-  - markerB
+  line:
+    - markerA
+    - markerB
 ```
 then the order of tracking will be `markerA` followed by `markerB`.
 
 %---------------------------------------------------------------------------------------------------
 (s:superposition)=
 ## Superposition
 
-In Construction...
+The superposition construct is used to add elements to a beamline after the beamline has been defined.
+This allows, for example, modifications to the lattice created from an existing lattice file,
+without having to modify the existing lattice file itself, by creating a new file that 
+[includes](#s:includefiles) the original file and then uses superposition to modify the lattice.
+
+A superposition specifies an element to `insert` and a [`placement`](#s:placement) construct
```

Comment:
It might be helpful to use an alternative word like `superimpose` in place of `insert`.